bindings/python/triplestore/triplestore.py

```diff
@@ -0,0 +1,608 @@
+'''A module encapsulating different triplestores using the strategy design
+pattern.
+
+This module has no dependencies outside the standard library, but the
+triplestore backends have.
```

Comment:
Is this critical to mention here? Probably should be moved to a top-level README?

Reply:
Agree, it does not really belong together with the rest of the module documentation. But I still think it is a useful comment for the developer. 

If we move the module docstring to the `__init__.py` file, we could leave this comment where it is. Otherwise we could make it a separate comment. Placing it in the README file is also a good option...